### PR TITLE
docs: correct minor problems with the guide to migrating global filters

### PR DESCRIPTION
**Pull request by skirtles-code**
A few tweaks to the new guide to migrating global filters.

I think it's mostly self-explanatory but I've made review comments inline just in case.

### CODE REVIEW

File: src/guide/migration/filters.md [hunk 1]
```diff
@@ -75,17 +75,17 @@ Instead of using filters, we recommend replacing them with computed properties o
 
 ### Global Filters
 
-If you are using filters that were globally registered and then used throughout our app, it's likely not convenient to replace them with computed props or methods in each individual component.
+If you are using filters that were globally registered and then used throughout your app, it's likely not convenient to replace them with computed properties or methods in each individual component.
```
Comment (skirtles-code) [inline src/guide/migration/filters.md:78]: The sentence starts off with 'you', so I've updated the 'our' to 'your'.

'Computed properties' needs to be written out in full.

File: src/guide/migration/filters.md [hunk 2]
```diff
@@ -75,17 +75,17 @@ Instead of using filters, we recommend replacing them with computed properties o
...
 
-Instead, you can make your global filters available to all components through `app.globalProperties`:
+Instead, you can make your global filters available to all components through [globalProperties](../../api/application-config.html#globalproperties):
```
Comment (skirtles-code) [inline src/guide/migration/filters.md:80]: This either needs to be `app.config.globalProperties` or just `globalProperties`. I've gone with the latter and made it a link.

File: src/guide/migration/filters.md [hunk 3]
```diff
@@ -75,17 +75,17 @@ Instead of using filters, we recommend replacing them with computed properties o
...
 
 ```javascript
 // main.js
 const app = createApp(App)
 
 app.config.globalProperties.$filters = {
-  accountInUSD(num) {
-    return '$' + num
+  currencyUSD(value) {
```
Comment (skirtles-code) [inline src/guide/migration/filters.md:87]: For consistency with the Vue 2 example, I've changed the names to `currencyUSD`, `value` and `accountBalance`.

File: src/guide/migration/filters.md (hunk 1 above)
Comment (bencodezen) [reply src/guide/migration/filters.md:78]: Good improvement!

**Review by bencodezen — APPROVED** [review]
Nice improvements. Thank you @skirtles-code!

**Pull request merged**